src/app/components/Curation/getComponentName/index.ts

```diff
@@ -21,12 +22,19 @@ const {
   HIERARCHICAL_CURATION_GRID,
   MOST_READ,
   NOT_SUPPORTED,
+  RADIO_SCHEDULE
 } = COMPONENT_NAMES;
 
 export default (
   visualStyle: VisualStyle,
   visualProminence: VisualProminence,
+  curationType = ''
```

Comment:
Given that this function now takes more than 2 params, I want to pass the params by reference, and not by value.

By this I mean it's no longer
`getComponentName(thing1WhichHappensToBeAVisualStyle, thing2IsAVisualProminence)`
but `getComponentName({visualStyle, visualProminence, curationType})`

These props actually fit the `Curation` interface quite nicely, so we don't need to declare a new interface for this

```suggestion
export default ({
  visualStyle,
  visualProminence,
  curationType = ''
  }: Curation) 
```